atlasdb-commons/src/main/java/com/palantir/util/UniqueIds.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * (c) Copyright 2022 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.util;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.palantir.logsafe.Preconditions;
+import java.util.Random;
+import java.util.UUID;
+import java.util.concurrent.ThreadLocalRandom;
+
+public final class UniqueIds {
+    private UniqueIds() {}
+
+    public static UUID uuid() {
+        return uuid(ThreadLocalRandom.current());
+    }
+
```

Comment:
- [ ] Add v6, v7, v8 per https://www.ietf.org/archive/id/draft-peabody-dispatch-new-uuid-format-04.html

Reply:
deferred to future once spec is final